### DISCUSSION TITLE
Token Login

### DISCUSSION THREAD

**Discussion by ghost** [japandotorg/LemonCord#8] [Ideas]
An idea for a feature could be to allow token login.
For logging in via the UI. That could be difficult as you would need to inject HTML / JavaScript, I'm not aware of how to do that in wry.

A easier solution could be using .env files, Automatically login or prompt on startup.

Status: 
Issue isn't with discord. I just can't add add to window, Possibly contained javascript?

**Comment by japandotorg** [comment]
That's actually a decent idea and shouldn't be that hard on implementing it since there's a [real handy method in wry to inject javascript](https://docs.rs/wry/latest/wry/webview/struct.WebView.html#method.evaluate_script) and works the same way as the injecting method in other webview library written in different languages.

**Reply by ghost** [reply]
Interesting. Didn't know that, Should I work on implementing that now or is that more of a further down the line thing?

**Reply by japandotorg** [reply]
Doesn't really matters if it gets implemented right now or later, the sooner the better I guess?

**Reply by ghost** [reply]
> Doesn't really matters if it gets implemented right now or later, the sooner the better I guess?

Ah okay. I'll work on it.
